hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystem.java

```diff
@@ -700,6 +698,18 @@ public synchronized void close() throws IOException {
     if (isClosed) {
       return;
     }
+    if (abfsStore.getClient().isMetricCollectionEnabled()) {
+      TracingContext tracingMetricContext = new TracingContext(
+              clientCorrelationId,
+              fileSystemId, FSOperationType.GET_ATTR, true,
```

Comment:
Here for metric tracing context we are using fileSystemId only, where as the metric tracing context created in ABfsClient, we are suing hostname...

Any reason for this difference?

Reply:
Looks good this way only.